src/style.css

```diff
@@ -0,0 +1,131 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --dark-grey: #060b35;
+  --light-grey: #616070;
+  --blue: #00acdc;
+  --white: #fff;
+}
+
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container__card {
+  padding: 16px;
+}
+
+.card {
```

Comment:
Create separate files for styling `card` and `stars` blocks
![image](https://user-images.githubusercontent.com/96409563/226444860-ab8d0200-139d-4f55-9fd1-57f909621c14.png)